Runtime/Common/GuidHelper.cs

```diff
@@ -17,5 +17,11 @@ public static Guid FromLong(long source)
             Array.Copy(BitConverter.GetBytes(source), guidData, 8);
             return new Guid(guidData);
         }
+
+        public static bool IsEmptyGuid(string guid)
```

Comment:
Method should be `isEmptyOrNullOrEmptyGuid`.

Reply:
The class name is GuidHelper - the domain of the method is clear, but your suggestion anyway is really good. I'm going to rename this method to stick to .NET patterns and use something like "IsNullOrEmpty"